climpred/metrics.py

```diff
@@ -990,398 +1742,481 @@ def _msss_murphy(forecast, reference, dim=None, **metric_kwargs):
-          North Atlantic Multidecadal Variability.” Climate Dynamics 13,
-          no. 7–8 (August 1, 1997): 459–87. https://doi.org/10/ch4kc4.
-    """
-    mse_skill = __mse.function(forecast, reference, dim=dim, **metric_kwargs)
-    var = reference.var(dim)
-    if 'comparison' in metric_kwargs:
-        comparison = metric_kwargs['comparison']
-    else:
-        raise ValueError(
-            'Comparison needed to normalize NMSE. Not found in', metric_kwargs
-        )
-    fac = _get_norm_factor(comparison)
-    nmse_skill = mse_skill / var / fac
-    return nmse_skill
-
-
-__nmse = Metric(
-    name='nmse',
-    function=_nmse,
-    positive=False,
-    probabilistic=False,
-    unit_power=0,
-    long_name='Normalized Mean Squared Error',
-    aliases=['nev'],
-    minimum=0.0,
-    maximum=1.0,
-    perfect=0.0,
-)
+        * Matheson, James E., and Robert L. Winkler. “Scoring Rules for
+          Continuous Probability Distributions.” Management Science 22, no. 10
+          (June 1, 1976): 1087–96. https://doi.org/10/cwwt4g.
+        * Gneiting, Tilmann, and Adrian E Raftery. “Strictly Proper Scoring
+          Rules, Prediction, and Estimation.” Journal of the American
+          Statistical Association 102, no. 477 (March 1, 2007): 359–78.
+          https://doi.org/10/c6758w.
 
+    Example:
+        >>> compute_perfect_model(ds, control, metric='crpss')
+        >>> compute_perfect_model(ds, control, metric='crpss', gaussian=False,
+                                  cdf_or_dist=scipy.stats.norm, xminimum=-10,
+                                  xmaximum=10, tol=1e-6)
 
-def _nmae(forecast, reference, dim=None, **metric_kwargs):
+    See also:
+        * properscoring.crps_ensemble
+        * xskillscore.crps_ensemble
     """
-    Normalized Ensemble Mean Absolute Error metric.
-
-    .. math:: NMAE = \\frac{MAE}{\\sigma_{o} \\cdot fac}
+    # available climpred dimensions to take mean and std over
+    rdim = [tdim for tdim in reference.dims if tdim in CLIMPRED_DIMS]
+    mu = reference.mean(rdim)
+    sig = reference.std(rdim)
 
-    Args:
-        * forecast (xr.object)
-        * reference (xr.object)
-        * dim (str): dimension to apply metric to
-        * comparison (str): name comparison needed for normalization factor `fac`, see
-            :py:func:`climpred.metrics._get_norm_factor`
-            (internally required to be added via **metric_kwargs)
-        * metric_kwargs: (optional) weights, skipna, see xskillscore.mae
+    # checking metric_kwargs, if not found use defaults: gaussian, else crps_quadrature
+    if 'gaussian' in metric_kwargs:
+        gaussian = metric_kwargs['gaussian']
+    else:
+        gaussian = True
 
+    if gaussian:
+        ref_skill = _crps_gaussian(forecast, mu, sig)
+    else:
+        if 'cdf_or_dist' in metric_kwargs:
+            cdf_or_dist = metric_kwargs['cdf_or_dist']
+        else:
+            # TODO: Where is `norm` assigned??
+            cdf_or_dist = norm
```

Comment:
@aaronspring, see here. It looks like it's not being tested per coveralls, but I assume an error would be thrown here since nothing named `norm` is assigned.

Reply:
should be `scipy.stats.norm`. I think there are suppressed tests for this (maybe in xskillscore) because this takes ages

Reply:
Ah didn't see it as an import. Thanks.